misc-scripts/xref_mapping/XrefParser/VGNCParser.pm

```diff
@@ -32,7 +32,7 @@ XrefParser::VGNCParser
 A parser class to parse the VGNC source.
 VGNC is the official naming source for some vertebrates species
 
--data_uri = ftp://ftp.ebi.ac.uk/pub/databases/genenames/vgnc/tsv/vgnc_gene_set_All.txt.gz
```

Comment:
@TamaraNaboulsi @veidenberg 
This URL does not work, whereas the HTTPS one does.
How could the Xref work? Is it a new/recent change (I obviously missed)?

Reply:
Yes, many ftp URLs (including ones hosted by Ensembl) have been migrated to be accessible over https, because web browsers have dropped support for ftp:// protocol. The ftp:// version should still be accessible with an FTP client.

Reply:
Hi both. The VGNC link whcih xref uses to download the file is set to the correct http one, as you can see [here](https://github.com/Ensembl/ensembl-production/blob/main/modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/xref_sources.json#L207). It seems like this comment in VGNCParser.pm wasn't updated when the link was fixed. I will make sure to fix these comments with the next xref changes.